### DISCUSSION TITLE
Is pg_cron in Supabase support scheduled task for seconds?

### DISCUSSION THREAD

**Discussion by AndyYuGit** [supabase/supabase#18274] [Questions]
`-- Call a stored procedure every 5 seconds
SELECT cron.schedule('process-updates', '5 seconds', 'CALL process_updates()');`
I found this code snippet from the [document of this library in github ](https://github.com/citusdata/pg_cron) , so run job in second should be  possible by this library. However '5 seconds' expression seems not working in Supabase.

**Comment by GaryAustin1** [comment]
Supabase had an issue with the newer version and backed it out...
https://github.com/supabase/postgres/pull/652

**Reply by AndyYuGit** [reply]
May I know the best practice of running sub minute scheduled job , with the absence of above expression support currently in Supabase? Procedure with internal commit, edge function, or dblink? Thank you.

**Reply by GaryAustin1** [reply]
I've never tried to run sub minute tasks so have no advice.
There seem to be hits on the web for postgres...
https://dba.stackexchange.com/questions/245296/how-can-i-run-a-pgagent-job-every-10-second
